Documentation/ceph-cluster-crd.md

```diff
@@ -1382,6 +1382,7 @@ The `security` section contains settings related to encryption of the cluster.
   * `kms`: Key Management System settings
     * `connectionDetails`: the list of parameters representing kms connection details
     * `tokenSecretName`: the name of the Kubernetes Secret containing the kms authentication token
+    * `csiConnectionDetailsCMName`: the name of the Config Map containing the connection details as well as the token to use encrypted PVs from Ceph-CSI
```

Comment:
By *token*, do you mean the actual contents of the token, or the name of the Kubernetes Secret that contains the token?

Reply:
The name of the Secret containing the Token. Ceph-CSI can read it right?

Reply:
Yes, that is the plan. There might be a global/shared Secret that contains the Vault Token, and/or a Secret per tenant (Kubernetes Namespace) that Ceph-CSI is going to fetch.